tao/composer.json

```diff
@@ -1,11 +1,10 @@
 {
     "name": "oat-sa/pci-package",
+    "version": "1.0.0",
     "license": "GPL-2.0-only",
     "config": {
         "preferred-install": "dist",
-        "platform": {
-            "php": "8"
-        },
+        "platform": {},
```

Comment:
remove PHP version as it is redundant with the TAO extensions and create issues.